src/odemis/util/spot.py

```diff
@@ -373,27 +373,65 @@ def EstimateLatticeConstant(pos):
     return lattice_constants
 
 
-def GridPoints(grid_size_x, grid_size_y):
+def GridPointsLegacy(grid_size_x, grid_size_y):
```

Comment:
I'd rather not have 2 functions which probably do the same thing.
Especially, it seems the "legacy" version is used at  1 place, and the new function... nowhere.

I agree that in some cases, it's good to keep the previous version for a transition period. But not here, when in the entire code there is only one line using the function.